servers/src/main/java/tachyon/worker/block/evictor/EvictorType.java

```diff
@@ -30,6 +30,13 @@
    */
   LRU,
   /**
+   * Evict old blocks in certain StorageDir by LRU. The main difference between PartialLRU and
+   * LRU is that LRU choose old blocks among several StorageDirs until one StorageDir
+   * satisfies the request space, but PartialLRU select one StorageDir first and evict old
+   * blocks in certain StorageDir by LRU
+   */
+  PARTIAL_LRU,
```

Comment:
@Yours-shupeng oh, I understand it through codes, so, PartialLRU means we first select StorageDir with the maximum possible to be available bytes after eviction, and then select blocks in this dir with LRU.